ts/src/bitmex.ts

```diff
@@ -1591,9 +1738,11 @@ export default class bitmex extends Exchange {
         //         "timestamp": "2019-03-05T12:47:02.762Z"
         //     }
         //
+        const marketId = this.safeString (trade, 'symbol');
+        const symbol = this.safeSymbol (marketId, market);
         const timestamp = this.parse8601 (this.safeString (trade, 'timestamp'));
         const priceString = this.safeString2 (trade, 'avgPx', 'price');
-        const amountString = this.safeString2 (trade, 'size', 'lastQty');
+        const amountString = this.convertFromRawQuantity (symbol, this.safeString2 (trade, 'size', 'lastQty'));
```

Comment:
I also suggest to look below 1720-th line:

https://github.com/ccxt/ccxt/pull/15311/files#diff-c67aea398a91c3fcd4bba7547d89b0fc53da95e0112d21d4a5ff084a738fba74R1728